invokeai/app/invocations/image.py

```diff
@@ -59,6 +60,19 @@ class Config:
             ]
         }
 
+class ImageMaskOutput(BaseInvocationOutput):
```

Comment:
Could you add the this to the list of base nodes in the docs?

https://github.com/invoke-ai/InvokeAI/blob/main/docs/features/NODES.md

Reply:
Sure, I will make a PR to add FaceMask to the nodes docs.